### DISCUSSION TITLE
Admin dashboard marketplace

### DISCUSSION THREAD

**Discussion by amirping** [medusajs/medusa#4585] [Q&A] [answered]
My question regarding the admin dashboard is, does it support multiple stores per user and multiple users with different stores (marketplace).

**Comment by edardev** [comment, marked as answer]
Medusa only supports 1 store by default, although you can customize it by following this tutorial example for a "marketplace": https://medusajs.com/blog/extending-medusa-usecase-marketplace

I wish it could support unlimited stores by default for a project I am planning to build so it can be easier to upgrade to future versions without breaking code changes, I wonder if we could use multiple "Regions" as individual stores so products and customers can be assigned to specify "region as a store" without having to update the code to support multiple Stores.

We are looking for suggestions about this feature.
